### PR TITLE
document deployment process

### PR DESCRIPTION
**Pull request by turadg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # dapp-psm
+
 UI for Inter Protocol PSM
+
+## Development
+
+`yarn dev` to start a local HMR server.
+
+By default it it will connect to the prod Wallet UI. To connect to a local one, use
+http://127.0.0.1:5173/?wallet=local .
```
Comment (samsiegart) [inline README.md:10]: It also works with different deployments of `wallet-app` for testing. I.e. `https://<branch-name>.wallet-app.pages.dev` can be used with `?wallet=<branch-name>`

File: tailwind.config.cjs [hunk 2]
```diff
@@ -1,8 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   mode: 'jit', // update this line
-  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
-  darkMode: false,
```
Comment (samsiegart) [inline tailwind.config.cjs:5]: What did this do?

Comment (turadg) [reply tailwind.config.cjs:5]: Obsolete with Tailwind 3.0. The console kept warning about it.

File: README.md (hunk 1 above)
Comment (turadg) [reply README.md:10]: yeah. I figured that's advanced mode. Feel free to amend the commit or PR to include

**Review by samsiegart — APPROVED** [review]

**Pull request merged**